tests/vocabularies_tests.py

```diff
@@ -100,5 +100,17 @@ def test_get_rightsinfo(self):
             self.assertEqual('foo notice', info['copyrightnotice'])
             self.assertEqual('foo terms', info['usageterms'])
 
+    def test_get_locale_vocabulary(self):
+        items = [
+            {'is_active': True, 'name': 'FIXME1', 'qcode': 'f', 'subject': '',
+             'translations': {'name': {'fr': 'FIXME1-fr', 'es': 'FIXME1-es'}}},
+            {'is_active': True, 'name': 'FIXME2', 'qcode': 'f', 'subject': '',
+             'translations': {'name': {'fr': 'FIXME2-fr', 'es': 'FIXME2-es'}}}
+        ]
+        result = VocabulariesService().get_locale_vocabulary(items, 'fr')
+
+        self.assertEqual(result[0]['name'], 'FIXME1-fr')
```

Comment:
can you check it like `self.assertEqual(result[0], {'name': 'FIXME1-fr', 'code': 'f'})` so we make sure there are no extra fields like translations

Reply:
Why we have to drop translations since it is already on original vocabulary?

Reply:
because it shouldn't make it to the output

Reply:
I think it is not the case, the code that uses vocabularies just pick the name or any other value that is needed. The vocabulary is not put in output as a whole.

Reply:
you're right, it's not used directly.